docs/install.rst

```diff
@@ -18,39 +18,71 @@ Requirements
 PlasmaPy requires Python version 3.7 or newer.
 PlasmaPy requires the following packages for installation:
 
-- `NumPy <http://www.numpy.org/>`_ 1.18.1 or newer
-- `SciPy <https://www.scipy.org/>`_ 1.2 or newer
-- `Astropy <http://www.astropy.org/>`_ 4.0 or newer
+- `NumPy <https://www.numpy.org/>`_ — 1.18.1 or newer
+- `SciPy <https://www.scipy.org/>`_ — 1.2 or newer
+- `Astropy <https://www.astropy.org/>`_ — 4.0 or newer
+- `pandas <https://pandas.pydata.org/>`_ — 1.0 or newer
+- `xarray <http://xarray.pydata.org>`_ — above 0.14 but less than 0.16
```

Comment:
```suggestion
- `xarray <http://xarray.pydata.org>`_ — 0.14 and above, but less than 0.17
```

Reply:
This is no longer true, I think we fixed the limit < 0.17 before in grids 4 or so. Will double check.

Reply:
We fixed it and added tests for xarray-dev but forgot to remove the requirement limit. My bad.